### PR TITLE
@katyhuff edits implemented

### PR DESCRIPTION
**Pull request by andrewryh**
This PR resolves issues #5 #8 #9 #10 #12 #13 #14 #16 #17

### CODE REVIEW

File: abstract.tex [hunk 1]
```diff
@@ -1,24 +1,22 @@
...
-reactivity (5500 pcm). Finally, the average $^{232}$Th feed rate throughout 60 
-years of operation is 100 g/GWh$_e$ which is a good agreement with other recent 
-research. Problematic effects of neutron energy spectrum hardening during 
-\gls{MSBR} operation should be taken into account for neutronics, 
-multi-physics, and fuel cycle performance analysis.
+Nowadays climate change pushes humankind in search new ways to generate 
+carbon-free, reliable, nonintermittent base load power. That is reason why 
+interest in advanced nuclear energy and, particularly \glspl{MSR} has resurged, 
+with multiple new companies pursuing commercialization of \gls{MSR} 
+designs\footnote{Examples include both liquid-fueled and solid-fueled molten
```
Comment (katyhuff) [inline abstract.tex]: In the abstract, this should be a sentence rather than a footnote.

File: abstract.tex [hunk 2]
```diff
@@ -1,24 +1,22 @@
...
-problematic impact on safety parameters (e.g., temperature reactivity feedback, 
-reactivity control system worth). Volatile gases and noble metals removal has 
-positive effect on the core performance approximately 7500 pcm over 10-years 
-timeframe; rare earth elements removal has slower rate and less impact on 
-reactivity (5500 pcm). Finally, the average $^{232}$Th feed rate throughout 60 
-years of operation is 100 g/GWh$_e$ which is a good agreement with other recent 
-research. Problematic effects of neutron energy spectrum hardening during 
-\gls{MSBR} operation should be taken into account for neutronics, 
-multi-physics, and fuel cycle performance analysis.
+Nowadays climate change pushes humankind in search new ways to generate
```
Comment (katyhuff) [inline abstract.tex]: "Nowadays" is too colloquial.

File: abstract.tex [hunk 3]
```diff
@@ -1,24 +1,22 @@
...
-reactivity control system worth). Volatile gases and noble metals removal has 
-positive effect on the core performance approximately 7500 pcm over 10-years 
-timeframe; rare earth elements removal has slower rate and less impact on 
-reactivity (5500 pcm). Finally, the average $^{232}$Th feed rate throughout 60 
-years of operation is 100 g/GWh$_e$ which is a good agreement with other recent 
-research. Problematic effects of neutron energy spectrum hardening during 
-\gls{MSBR} operation should be taken into account for neutronics, 
-multi-physics, and fuel cycle performance analysis.
+Nowadays climate change pushes humankind in search new ways to generate 
+carbon-free, reliable, nonintermittent base load power. That is reason why
```
Comment (katyhuff) [inline abstract.tex]: nonintermittent should be deleted.

File: abstract.tex [hunk 4]
```diff
@@ -1,24 +1,22 @@
...
-positive effect on the core performance approximately 7500 pcm over 10-years 
-timeframe; rare earth elements removal has slower rate and less impact on 
-reactivity (5500 pcm). Finally, the average $^{232}$Th feed rate throughout 60 
-years of operation is 100 g/GWh$_e$ which is a good agreement with other recent 
-research. Problematic effects of neutron energy spectrum hardening during 
-\gls{MSBR} operation should be taken into account for neutronics, 
-multi-physics, and fuel cycle performance analysis.
+Nowadays climate change pushes humankind in search new ways to generate 
+carbon-free, reliable, nonintermittent base load power. That is reason why 
+interest in advanced nuclear energy and, particularly \glspl{MSR} has resurged,
```
Comment (katyhuff) [inline abstract.tex]: delete the comma between "and" and "particularly"

File: abstract.tex (hunk 3 above)
Comment (katyhuff) [reply abstract.tex]: "That is reason why" should be "Thus,"

File: abstract.tex [hunk 5]
```diff
@@ -1,24 +1,22 @@
...
-\gls{MSBR} operation should be taken into account for neutronics, 
-multi-physics, and fuel cycle performance analysis.
+Nowadays climate change pushes humankind in search new ways to generate 
+carbon-free, reliable, nonintermittent base load power. That is reason why 
+interest in advanced nuclear energy and, particularly \glspl{MSR} has resurged, 
+with multiple new companies pursuing commercialization of \gls{MSR} 
+designs\footnote{Examples include both liquid-fueled and solid-fueled molten 
+salt designs from
+Transatomic, Terrapower, Terrestrial, Moltex Energy, and Thorcon.}. To further 
+develop these \gls{MSR} concepts researchers need tools for analyzing liquid
```
Comment (katyhuff) [inline abstract.tex]: comma after concepts.

tools -> simulation tools

File: abstract.tex [hunk 6]
```diff
@@ -1,24 +1,22 @@
...
+interest in advanced nuclear energy and, particularly \glspl{MSR} has resurged, 
+with multiple new companies pursuing commercialization of \gls{MSR} 
+designs\footnote{Examples include both liquid-fueled and solid-fueled molten 
+salt designs from
+Transatomic, Terrapower, Terrestrial, Moltex Energy, and Thorcon.}. To further 
+develop these \gls{MSR} concepts researchers need tools for analyzing liquid 
+fueled molten salt reactor depletion and fuel processing. However, most 
+contemporary nuclear reactor physics software is unable to perform high-fidelity 
+full-core depletion calculations in an online reprocessing regime. This paper 
+introduces a Python script, SaltProc, which couples with the Monte Carlo code,
```
Comment (katyhuff) [inline abstract.tex]: script should be "package"

File: abstract.tex [hunk 7]
```diff
@@ -1,24 +1,22 @@
...
+designs\footnote{Examples include both liquid-fueled and solid-fueled molten 
+salt designs from
+Transatomic, Terrapower, Terrestrial, Moltex Energy, and Thorcon.}. To further 
+develop these \gls{MSR} concepts researchers need tools for analyzing liquid 
+fueled molten salt reactor depletion and fuel processing. However, most 
+contemporary nuclear reactor physics software is unable to perform high-fidelity 
+full-core depletion calculations in an online reprocessing regime. This paper 
+introduces a Python script, SaltProc, which couples with the Monte Carlo code, 
+SERPENT2, and simulates \gls{MSR} online reprocessing by modeling the changing 
+isotopic composition of an irradiated fuel salt. SaltProc capabilities was
```
Comment (katyhuff) [inline abstract.tex]: get rid of irradiated. 
Perhaps "changing isotopic composition of MSR fuel salt"


Also "SaltProc capabilities were demonstrated"

File: abstract.tex [hunk 8]
```diff
@@ -1,24 +1,22 @@
...
+salt designs from
+Transatomic, Terrapower, Terrestrial, Moltex Energy, and Thorcon.}. To further 
+develop these \gls{MSR} concepts researchers need tools for analyzing liquid 
+fueled molten salt reactor depletion and fuel processing. However, most 
+contemporary nuclear reactor physics software is unable to perform high-fidelity 
+full-core depletion calculations in an online reprocessing regime. This paper 
+introduces a Python script, SaltProc, which couples with the Monte Carlo code, 
+SERPENT2, and simulates \gls{MSR} online reprocessing by modeling the changing 
+isotopic composition of an irradiated fuel salt. SaltProc capabilities was 
+demonstrated for full-core high-fidelity model of the commercial \gls{MSBR}
```
Comment (katyhuff) [inline abstract.tex]: "for a full-core, high-fidelity model"

File: introduction.tex [hunk 9]
```diff
@@ -1,92 +1,90 @@
...
+fuel cycle, which uses natural thorium instead of enriched uranium. With 
+continuous fuel reprocessing, the \gls{MSBR} realizes the advantages of the 
+thorium fuel cycle because the $^{233}$U bred from 
+$^{232}$Th is almost instantly \footnote{\space $^{232}$Th transmutes into 
+$^{233}$Th after capturing a neutron. Next, this isotope decays to $^{233}$Pa 
+($\tau_{1/2}$=21.83m), which finally decays to $^{233}$U 
+($\tau_{1/2}$=26.967d).} recycled back to the core 
+\cite{betzler_modeling_2016}. The chosen fuel salt, LiF-BeF$_2$-ThF$_4$-UF$_4$, has a 
+melting point of $499^\circ$C, a low vapor pressure at operating temperatures, 
+and good flow and heat transfer properties \cite{robertson_conceptual_1971}. In
```
Comment (katyhuff) [inline introduction.tex:34]: "good" is still vague. But ok.

File: introduction.tex [hunk 10]
```diff
@@ -130,11 +139,11 @@ \section{Introduction}
 \cite{nuttin_potential_2005}.  A similar approach was used by Powers \emph{et 
 al.}, Betzler \emph{et al.}, and Jeong \emph{et al.} 
 \cite{powers_new_2013,powers_inventory_2014,betzler_modeling_2016, 
-betzler_molten_2017, jeong_development_2014, jeong_equilibrium_2016}. This 
-approach misrepresents the two-region breeder reactor concept. The unit-cell or 
-one-region models may produce reliable results for homogeneous reactor cores 
-(i.e. \gls{MSFR}, \gls{MOSART}) or for one-region single-fluid reactor designs 
-(i.e. \gls{MSRE}). However, a two-region \gls{MSBR} must be simulated using a whole-core 
+betzler_molten_2017, jeong_development_2014, jeong_equilibrium_2016}. This approach 
+misrepresent the two-region breeder reactor concept. The unit-cell or one-region
```
Comment (katyhuff) [inline introduction.tex]: misrepresents

File: results.tex [hunk 11]
```diff
@@ -57,15 +56,15 @@ \subsection{Effective multiplication factor}
...
-depletion time step. The concentrations of $^{233}$U, $^{232}$Th, $^{233}$Pa, and 
-$^{232}$Pa in fuel salt change insignificantly after approximately 2500 days of 
-operation. Particularly, $^{233}$U number density fluctuates less than 0.8\% in 
-the time interval from 16 to 20 years of operation. Hence, a quasi-equilibrium 
-state was achieved after 16 years of reactor operation.
-\begin{figure}[ht!] % replace 't' with 'b' to \centering
+information about the equilibrium state. Figure~\ref{fig:adens_eq} shows number 
+density of major nuclides which have a strong influence on the reactor core 
+physics. The concentration of $^{233}$U, $^{232}$Th, $^{233}$Pa, and $^{232}$Pa in 
+fuel salt change insignificantly after approximately 2500 days of operation.
```
Comment (katyhuff) [inline results.tex]: the fuel salt

File: results.tex [hunk 12]
```diff
@@ -57,15 +56,15 @@ \subsection{Effective multiplication factor}
...
-$^{232}$Pa in fuel salt change insignificantly after approximately 2500 days of 
-operation. Particularly, $^{233}$U number density fluctuates less than 0.8\% in 
-the time interval from 16 to 20 years of operation. Hence, a quasi-equilibrium 
-state was achieved after 16 years of reactor operation.
-\begin{figure}[ht!] % replace 't' with 'b' to \centering
+information about the equilibrium state. Figure~\ref{fig:adens_eq} shows number 
+density of major nuclides which have a strong influence on the reactor core 
+physics. The concentration of $^{233}$U, $^{232}$Th, $^{233}$Pa, and $^{232}$Pa in 
+fuel salt change insignificantly after approximately 2500 days of operation. 
+Particularly, $^{233}$U number density fluctuates less than 0.8\% in the time
```
Comment (katyhuff) [inline results.tex]: "the ${233}U$"

File: results.tex [hunk 13]
```diff
@@ -358,12 +360,13 @@ \subsection{The effect of removing fission product from fuel salt}
...
 subcriticality ($k_{eff}<$1.0) for full-core model) called the core lifetime. 
-Therefore, removing fission products provides with significant neutronic 
-benefit because allows achieve longer core lifetime.
-\begin{figure}[ht!] % replace 't' with 'b' to force it to \centering
-  \includegraphics[width=\textwidth]{keff_rem_cases.png} \caption{Calculated 
-  effective multiplication factor for full-core \gls{MSBR} model with removal 
-  of various fission product groups over 10 years of operation. The 
-  confidence interval $\pm\sigma	$ is shaded.}
+Therefore, removing fission products provides with significant neutronic benefit 
+because allows achieve longer core lifetime.
```
Comment (katyhuff) [inline results.tex]: "because allows achieve longer" should be "and enables a"

File: methods.tex [hunk 14]
```diff
@@ -1,14 +1,10 @@
...
-Liquid-fueled system modeling with contemporary reactor physics codes is 
-difficult because most of these codes were developed for analyzing solid-fueled 
-reactors. In fact, world-wide commercial reactors fleet today is represented by 
-systems with solid fuel. Liquid-fueled systems ability to continuously remove 
-fission products and add fissile and/or fertile elements is sthe main challenge 
-for depletion simulations. SaltProc takes into account online separations and 
-feeds using the SERPENT 2 continuous-energy Monte Carlo neutron transport and 
-depletion code.  All calculations presented in this study were performed using 
-SERPENT 2 version 2.1.30 on Blue Waters’ XE6 nodes. 
+Liquid-fueled systems ability to continuously remove fission products and add
```
Comment (katyhuff) [inline methods.tex]: "Liquid-fueled systems ability" should be "The ability of liquid-fueled systems".

File: methods.tex [hunk 15]
```diff
@@ -1,14 +1,10 @@
...
-reactors. In fact, world-wide commercial reactors fleet today is represented by 
-systems with solid fuel. Liquid-fueled systems ability to continuously remove 
-fission products and add fissile and/or fertile elements is sthe main challenge 
-for depletion simulations. SaltProc takes into account online separations and 
-feeds using the SERPENT 2 continuous-energy Monte Carlo neutron transport and 
-depletion code.  All calculations presented in this study were performed using 
-SERPENT 2 version 2.1.30 on Blue Waters’ XE6 nodes. 
+Liquid-fueled systems ability to continuously remove fission products and add 
+fissile and/or fertile elements is the main challenge for depletion simulations. 
+Introduced in this work Python-package, SaltProc, takes into account online
```
Comment (katyhuff) [inline methods.tex]: "Introduced in this work Python-package, SaltProc," should be "The python package introduced in this work, Salt Proc"

File: methods.tex [hunk 16]
```diff
@@ -1,14 +1,10 @@
...
-systems with solid fuel. Liquid-fueled systems ability to continuously remove 
-fission products and add fissile and/or fertile elements is sthe main challenge 
-for depletion simulations. SaltProc takes into account online separations and 
-feeds using the SERPENT 2 continuous-energy Monte Carlo neutron transport and 
-depletion code.  All calculations presented in this study were performed using 
-SERPENT 2 version 2.1.30 on Blue Waters’ XE6 nodes. 
+Liquid-fueled systems ability to continuously remove fission products and add 
+fissile and/or fertile elements is the main challenge for depletion simulations. 
+Introduced in this work Python-package, SaltProc, takes into account online 
+separations and feeds using SERPENT 2 continuous-energy Monte Carlo neutron
```
Comment (katyhuff) [inline methods.tex]: "using SERPENT 2" should be "using the SERPENT 2"

File: introduction.tex [hunk 17]
```diff
@@ -96,32 +94,43 @@ \section{Introduction}
...
-Python-based script for SCALE/TRITON which uses a semi-continuous batch process 
-to simulate a continuous reprocessing. ChemTriton models salt treatment, 
-separations, discharge, and refill using a unit-cell \gls{MSR} SCALE/TRITON 
-model over small time steps to simulate continuous reprocessing and deplete the 
-fuel salt \cite{powers_new_2013}.
+Accounting for continuous removal or addition presents a greater challenge since it 
+requires adding a term to the Bateman equations. In SCALE 
+\cite{bowman_scale_2011}, ORIGEN \cite{gauld_isotopic_2011} solves a set of 
+Bateman equations using spectrum-averaged fluxes and cross sections generated 
+from a deterministic transport calculation. Approaches listed in references
```
Comment (katyhuff) [inline introduction.tex]: "Approaches" should be "Meanwhile, approaches"

File: introduction.tex [hunk 18]
```diff
@@ -96,32 +94,43 @@ \section{Introduction}
...
+References 
+\cite{doligez_coupled_2014,heuer_towards_2014,sheu_depletion_2013,aufiero_extended_2013} 
+simulate some form of reactivity control, and methods 
+\cite{doligez_coupled_2014,heuer_towards_2014,aufiero_extended_2013,ahmad_neutronics_2015, 
+park_whole_2015, 
+jeong_equilibrium_2016,rykhlevskii_online_2017,nuttin_potential_2005} use a set 
+of all nuclides in depletion calculations. 
+
+Many liquid-fueled \gls{MSR} designs rely on online fuel processing in which  
+material moves to and from the core at specific time steps (batch) or
```
Comment (katyhuff) [inline introduction.tex]: "material moves to and from the core at specific time steps (batch) or continuously"
should be 
"material moves to and from the core continuously or at specific time steps (batch-wise)"

File: introduction.tex [hunk 19]
```diff
@@ -96,32 +94,43 @@ \section{Introduction}
...
+\cite{doligez_coupled_2014,heuer_towards_2014,sheu_depletion_2013,aufiero_extended_2013} 
+simulate some form of reactivity control, and methods 
+\cite{doligez_coupled_2014,heuer_towards_2014,aufiero_extended_2013,ahmad_neutronics_2015, 
+park_whole_2015, 
+jeong_equilibrium_2016,rykhlevskii_online_2017,nuttin_potential_2005} use a set 
+of all nuclides in depletion calculations. 
+
+Many liquid-fueled \gls{MSR} designs rely on online fuel processing in which  
+material moves to and from the core at specific time steps (batch) or 
+continuously. In batch-wise approach the burn-up simulation stops at a given
```
Comment (katyhuff) [inline introduction.tex]: "In batch-wise approach the" should be "In the batch-wise approach, the "

File: introduction.tex [hunk 20]
```diff
@@ -96,32 +94,43 @@ \section{Introduction}
...
+simulate some form of reactivity control, and methods 
+\cite{doligez_coupled_2014,heuer_towards_2014,aufiero_extended_2013,ahmad_neutronics_2015, 
+park_whole_2015, 
+jeong_equilibrium_2016,rykhlevskii_online_2017,nuttin_potential_2005} use a set 
+of all nuclides in depletion calculations. 
+
+Many liquid-fueled \gls{MSR} designs rely on online fuel processing in which  
+material moves to and from the core at specific time steps (batch) or 
+continuously. In batch-wise approach the burn-up simulation stops at a given 
+time and restart with a new liquid fuel composition (after removal of discarded
```
Comment (katyhuff) [inline introduction.tex]: "restart" should be "restarts"

File: introduction.tex [hunk 21]
```diff
@@ -96,32 +94,43 @@ \section{Introduction}
...
+time and restart with a new liquid fuel composition (after removal of discarded 
+materials and addition of fissile/fertile materials). \gls{ORNL} researchers 
+have developed ChemTriton, a Python-based script for SCALE/TRITON which uses the 
+batch-wise approach to simulate a continuous reprocessing. ChemTriton models salt 
+treatment, separations, discharge, and refill using a unit-cell \gls{MSR} 
+SCALE/TRITON depletion simulation over small time steps to simulate continuous 
+reprocessing and deplete the fuel salt \cite{powers_new_2013}. Methods listed in 
+references \cite{fiorina_investigation_2013,sheu_depletion_2013,park_whole_2015, 
+jeong_equilibrium_2016,powers_inventory_2014,betzler_molten_2017,rykhlevskii_online_2017} 
+and current work employ a batch-wise approach.
```
Comment (katyhuff) [inline introduction.tex]: "and current work" should be "as well as the current work"

**Review by katyhuff — CHANGES_REQUESTED** [review]
Thanks for making these changes. See comments for some minor grammatical issues.

**Comment by andrewryh** [comment]
@katyhuff Thanks for the comments. I've fixed it.

**Pull request merged**